dns_managed_zone_private_peering/main.tf

```diff
@@ -15,8 +15,12 @@
  */
 
 # [START dns_managed_zone_private_peering]
+resource "random_id" "zone_prefix" {
```

Comment:
value for `zone_prefix` is used as a suffix at end of zone name.  let's rename this resource to `zone_suffix`